sphinx_scality/static/css/current.css_t

```diff
@@ -77,7 +77,7 @@ div.sidebar {
     margin: 0;
     padding: 0 20px;
     width: auto;
-    overflow: auto;
+    overflow: visible;
```

Comment:
Overflow of a sidebar should not be visible, otherwise you'll get weird results if there's any and the sidebar is fixed (content is forever lost out of the screen).

Reply:
This is the solution we found to have "div.fixed-sidebar{
    position: sticky;"
overflow: auto is not compatible with the fixed right menu